src/Map.php

```diff
@@ -138,7 +146,15 @@ public function enabled($slug, $index)
         $index = 1 << ($index - 1);
         foreach (explode(static::DELIMITER, $slug) as $child) {
             $key = empty($key) ? $child : $key.static::DELIMITER.$child;
-            if (!isset($map[$key]) || !($map[$key] & $index)) {
+
+            $is_missing = !isset($map[$key]);
+            $is_disabled = $is_missing ?: !($map[$key] & $index);
+
+            if ($is_missing) {
```

Comment:
@slash28cu Tiny thing:  it looks like variables in the project are using `$camelCase` rather than `$snake_case`, could you update these for consistency?